src/Modules/Power.asm

```diff
@@ -38,88 +37,86 @@
+    mov                    Temp1,  #0FFh          ; Default full power
+    jb                     Flag_Startup_Phase,  set_pwm_limit_low_rpm_exit ; Exit if startup phase set
 
-    mov A, Low_Rpm_Pwr_Slope        ; Check if low RPM power protection is enabled
-    jz  set_pwm_limit_low_rpm_exit  ; Exit if disabled (zero)
+    mov                    A,  Low_Rpm_Pwr_Slope  ; Check if low RPM power protection is enabled
+    jz                     set_pwm_limit_low_rpm_exit ; Exit if disabled (zero)
 
-    mov A, Comm_Period4x_H
-    jz  set_pwm_limit_low_rpm_exit  ; Avoid divide by zero
+    mov                    A,  Comm_Period4x_H
+    jz                     set_pwm_limit_low_rpm_exit ; Avoid divide by zero
 
-    mov A, #255                 ; Divide 255 by Comm_Period4x_H
-    jnb Flag_Initial_Run_Phase, ($+5)   ; More protection for initial run phase
-    mov A, #127
-    mov B, Comm_Period4x_H
-    div AB
-    mov B, Low_Rpm_Pwr_Slope        ; Multiply by slope
-    mul AB
-    mov Temp1, A                    ; Set new limit
-    xch A, B
-    jz  ($+4)                   ; Limit to max
+    mov                    A,  #255               ; Divide 255 by Comm_Period4x_H
+    jnb                    Flag_Initial_Run_Phase,  ($+5) ; More protection for initial run phase
+    mov                    A,  #127
+    mov                    B,  Comm_Period4x_H
+    div                    AB
+    mov                    B,  Low_Rpm_Pwr_Slope  ; Multiply by slope
+    mul                    AB
+    mov                    Temp1,  A              ; Set new limit
+    xch                    A,  B
+    jz                     ($+4)                  ; Limit to max
 
-    mov Temp1, #0FFh
+    mov                    Temp1,  #0FFh
 
-    clr C
-    mov A, Temp1                    ; Limit to min
-    subb    A, Pwm_Limit_Beg
-    jnc set_pwm_limit_low_rpm_exit
+    clr                    C
+    mov                    A,  Temp1              ; Limit to min
+    subb                   A,  Pwm_Limit_Beg
+    jnc                    set_pwm_limit_low_rpm_exit
 
-    mov Temp1, Pwm_Limit_Beg
+    mov                    Temp1,  Pwm_Limit_Beg
 
 set_pwm_limit_low_rpm_exit:
-    mov Pwm_Limit_By_Rpm, Temp1
+    mov                    Pwm_Limit_By_Rpm,  Temp1
     ret
 
-
-;**** **** **** **** **** **** **** **** **** **** **** **** ****
-;
-; Set pwm limit high rpm
-;
-; Sets power limit for high rpm
-;
-;**** **** **** **** **** **** **** **** **** **** **** **** ****
+    ;**** **** **** **** **** **** **** **** **** **** **** **** ****
+    ;
+    ; Set pwm limit high rpm
+    ;
+    ; Sets power limit for high rpm
+    ;
+    ;**** **** **** **** **** **** **** **** **** **** **** **** ****
 set_pwm_limit_high_rpm:
-    clr C
-    mov A, Comm_Period4x_L
-IF MCU_TYPE >= 1
-    subb    A, #0A0h                    ; Limit Comm_Period4x to 160, which is ~510k erpm
+    clr                    C
+    mov                    A,  Comm_Period4x_L
+    IF MCU_TYPE >= 1
```

Comment:
Could IF and ENDIF preprocessor directives be kept with no identation?

Reply:
this IF directive is indented because it is inside another IF directive, you want the IF directives all non indented, but their content to be? I mean like so:

```
IF
    line1
    line 2
IF
        line3
        line4
ENDIF
ELSE
    line5
    line6
ENDIF
```

Reply:
I'd prefer all them to keep same indentation, the directives and the code:
```
IF
    line1
    line 2
IF
    line3
    line4
ENDIF
ELSE
    line5
    line6
ENDIF
``